src/platforms/common/configuration/options.mdx

```diff
@@ -854,19 +854,29 @@ Set this boolean to `false` to disable the scope sync from Java to NDK on Androi
 Set this boolean to `true` to automatically attach all threads to all logged events on Android.
 </ConfigKey>
 
-<ConfigKey name="enableAutoPerformanceTracking" supported={["react-native", "flutter"]}>
+<ConfigKey name="enableAutoPerformanceTracking" supported={["react-native"]}>
```

Comment:
@krystofwoldrich I noticed that a few breaking changes of the RN v5 docs are not reflected in the rest of the docs.
I suggest going thru https://docs.sentry.io/platforms/react-native/migration/#from-4x-to-5x and checking the usage of such breaking changes in the whole docs and making the fixes, similar to this PR.
Maybe create an issue for tracking this?

Reply:
Got it,
do you have some specific thing that you came across?

Reply:
I see, for example the `outOfMemoryTracking`

Reply:
Also `enableAutoPerformanceTracking`? maybe more, better to double-check everything that has changed, including the changes directly on the JS SDK.